src/components/CommentList/CommentList.jsx

```diff
@@ -1 +1,17 @@
-export const CommentList = () => <>Put the list here</>;
+import { CommentInfo } from '../CommentInfo/CommentInfo';
+import './CommentList.scss';
+
+export const CommentList = ({ comments }) =>
```

Comment:
Ensure that the `comments` prop is passed correctly to the `CommentList` component. It should be an array of comment objects, each with an `id` property.